From 4b3e904aadb7536585e7595ad01c070467851f35 Mon Sep 17 00:00:00 2001
From: Cyril Hrubis <chrubis@suse.cz>
Date: Thu, 21 Mar 2013 12:11:18 +0100
Subject: [PATCH] testcases/.../process_stress: Silence the output.

The testcase was priting "Test exiting with SUCCESS"
from each child which generated hundreds of useless
lines in the log.

This moves the printf into the main process and the
message is printed only once on the test exit.

Signed-off-by: Cyril Hrubis <chrubis@suse.cz>
---
 testcases/kernel/sched/process_stress/process.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/testcases/kernel/sched/process_stress/process.c b/testcases/kernel/sched/process_stress/process.c
index 4c91425..c8def97 100644
--- a/testcases/kernel/sched/process_stress/process.c
+++ b/testcases/kernel/sched/process_stress/process.c
@@ -1132,6 +1132,7 @@ void messenger(void)
 				prtln();
 				if (discrim) {
 					prtln();
+					printf("Test exiting with SUCCESS\n");
 					exit(0);
 				}
 				exit(1);
@@ -1204,7 +1205,6 @@ void doit(void)
 
 			}
 		}
-		printf("Test exiting with SUCCESS\n");
 		exit(0);
 	}
 #ifdef __64LDT__
-- 
1.7.8.6

